sites/main-site/src/content/events/2025/hackathon-march-2025/dkfz-and-ghga.md

```diff
@@ -0,0 +1,23 @@
+---
+title: 'Hackathon - March 2025 (The German Cancer Research Center and GHGA)'
+subtitle: 'Local node of the nf-core hackathon at The German Cancer Research Center and GHGA'
+shortTitle: 'The German Cancer Research Center and GHGA'
+type: 'hackathon'
+startDate: '2025-03-17'
+startTime: '09:00+01:00'
+endDate: '2025-03-19'
+endTime: '17:00+01:00'
+locations:
+  - name: The German Cancer Research Center and GHGA
```

Comment:
I think these are co-organizers not also sharing a location

Reply:
Fixed